pkg/audit/manager.go

```diff
@@ -743,16 +747,18 @@ func emitEvent(constraint *unstructured.Unstructured,
 	timestamp, enforcementAction, rkind, rnamespace, rname, message, gkNamespace string,
 	eventRecorder record.EventRecorder) {
 	annotations := map[string]string{
-		"process":                   "audit",
-		"auditTimestamp":            timestamp,
-		logging.EventType:           "violation_audited",
-		logging.ConstraintKind:      constraint.GetKind(),
-		logging.ConstraintName:      constraint.GetName(),
-		logging.ConstraintNamespace: constraint.GetNamespace(),
-		logging.ConstraintAction:    enforcementAction,
-		logging.ResourceKind:        rkind,
-		logging.ResourceNamespace:   rnamespace,
-		logging.ResourceName:        rname,
+		"process":                    "audit",
+		"auditTimestamp":             timestamp,
+		logging.EventType:            "violation_audited",
+		logging.ConstraintGroup:      constraint.GroupVersionKind().Group,
+		logging.ConstraintAPIVersion: constraint.GroupVersionKind().Version,
+		logging.ConstraintKind:       constraint.GetKind(),
+		logging.ConstraintName:       constraint.GetName(),
+		logging.ConstraintNamespace:  constraint.GetNamespace(),
+		logging.ConstraintAction:     enforcementAction,
+		logging.ResourceKind:         rkind,
```

Comment:
We should be logging resource group/version for the violating resource as well.

Reply:
Done